src/components/masters/program/main-content/data/tests/actions.test.js

```diff
@@ -0,0 +1,73 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+
+import {
+  FETCH_PROGRAM_COURSE_ENROLLMENTS_REQUEST,
+  FETCH_PROGRAM_COURSE_ENROLLMENTS_SUCCESS,
+  FETCH_PROGRAM_COURSE_ENROLLMENTS_FAILURE,
+  CLEAR_PROGRAM_COURSE_ENROLLMENTS,
+} from '../constants';
+import {
+  fetchProgramEnrollmentOverview,
+  clearProgramEnrollmentOverview,
+} from '../actions';
+import * as service from '../service';
+
+const mockStore = configureMockStore([thunk]);
+jest.mock('../service');
+
+describe('fetchProgramEnrollmentOverview action', () => {
```

Comment:
after renaming function variables throughout, make sure to update the names of the tests to reflect the new names.

Reply:
I believe I renamed everything appropriately including tests.